src/helpers/text_helper.ts

```diff
@@ -224,6 +224,40 @@ export function getContextFontSize(font: string): Pixel {
   return Number(font.match(pxRegex)?.[1]);
 }
 
+export function drawSplittedText(
+  context: CanvasRenderingContext2D,
+  text: string,
+  position: PixelPosition
+) {
+  const splitted = splitText(text);
+  const measure = context.measureText(splitted[1]);
+  context.fillText(splitted[1], position.x, position.y);
+  position.y -= measure.actualBoundingBoxAscent + measure.actualBoundingBoxDescent + 2;
+  context.fillText(splitted[0], position.x, position.y);
+}
+
+export function splitText(text: string): string[] {
```

Comment:
```suggestion
export function splitTextInTwoLines(text: string): string[] {
```
or something like that